### ISSUE TITLE
Create the tx page

### ISSUE THREAD

**Issue by SlnPons** [massalabs/station-massa-wallet#456]
**Context**
As soon as this feature is unblocked by the explorer team, we should add the transaction page to Massa Wallet

**User flow**

Design: https://www.figma.com/file/LSaOibVDFb1reF6FePVn5l/Massastation?type=design&node-id=324-5834&t=k6TgyAxWK4906Fzs-0

@longgamma-massa @LucasF243 I added a comment on the tx page for both of you to check and rework together as I think we can do better / have things to change. I'll let you get organise and check this when it's a good time :)

**How to**

- [ ] Create components 
- [ ] Integrate component in the FE (think about integrating a placeholder state as info coming from the blockchain may be slow)
- [ ] Create endpoint or if they already exist connect to block API to retrieve info we need / want from the blockchain
- [ ] Connect BE & FE together 

**Technical details**
*Give the technical insights so anyone in the team can tackle the tasks - Dev*

**QA testing**
Follow user flow

**Comment by longgamma-massa** [comment]
Blocked by the release of Public Indexer API. 

Potential way to go is:
1.  store history somewhere similarly to Storer, in the Station for some time. Can be just hashes to redirect to Block Explorer website as well.
2. show temporary data, up to node's memory. 

For me this feature is an industry standard of wallets. IMO, a simple solution would work for the beginning.
@gregLibert @peterjah @mazmassa  WDYT?

**Comment by gregLibert** [comment]
> Blocked by the release of Public Indexer API.
> 
> Potential way to go is:
> 
> 1. store history somewhere similarly to Storer, in the Station for some time. Can be just hashes to redirect to Block Explorer website as well.
> 2. show temporary data, up to node's memory.
> 
> For me this feature is an industry standard of wallets. IMO, a simple solution would work for the beginning. @gregLibert @peterjah @mazmassa WDYT?

I don't think that having a 5 minutes history is a good idea. This being said, it could be cool to have a way to follow pending transaction up to there final status...
